apps/dashboard/src/components/workflow-editor/steps/digest/numbers-picker.tsx

```diff
@@ -0,0 +1,135 @@
+import { KeyboardEventHandler, useMemo, useRef, useState } from 'react';
+import { RiCornerDownLeftLine } from 'react-icons/ri';
+import type { PopoverContentProps } from '@radix-ui/react-popover';
+
+import { Button } from '@/components/primitives/button';
+import { InputFieldPure } from '@/components/primitives/input';
+import { Popover, PopoverContent, PopoverPortal, PopoverTrigger } from '@/components/primitives/popover';
+import TruncatedText from '@/components/truncated-text';
+import { cn } from '@/utils/ui';
+
+const textClassName = 'text-foreground-600 text-xs font-medium px-2';
+
+export const NumbersPicker = <T extends string | number>({
```

Comment:
The `NumbersPicker` is the component that allows you to pick hours, minutes, and days of the month.